src/pages/Routes/Routes.jsx

```diff
@@ -18,6 +19,7 @@ function Routes() {
         <Route path="/find-matches" element={<Pairing />} />
         <Route path="/chat" element={<Chat />} />
         <Route path="/accounts" element={<Example />} />
+        <Route path="/settings" element={<Settings />} />
```

Comment:
is there a difference between accounts and settings? also would it be good idea to remove accounts if we using settings instead?

Reply:
I thought it was a placeholder for something that's to be added. Shall I remove it then?

Reply:
The `/settings` route has been removed from the router, as well as the `SideBar` component. The new screenshots have been added to the PR.